docs/how-tos/configure.md

```diff
@@ -42,18 +42,21 @@ Install it on the computer controlling your smart machine by following the {{< g
 
 If you are using a microcontroller instead of a 64-bit computer, you can install a [lightweight version of `viam-server`](/installation/#install-viam-micro-server).
 You can install `viam-server` on your personal computer, or on a single-board computer (SBC).
+After following the setup instructions and starting `viam-server` on your machine, your machine should appear as **Live** on the Viam app.
 
 {{% /tablestep %}}
 {{% tablestep link="/configure/" %}}
-**3. Navigate to the CONFIGURE page of your machine**
+**3. Navigate to the CONFIGURE tab of your machine**
 
-Machines can be small and simple or very complex but they are all configured on the **CONFIGURE** tab.
+Machines can be small and simple or very complex but they are all configured on the **CONFIGURE** tab of the [Viam app](https://app.viam.com/).
 A machine can be a single-board computer with a single sensor or LED wired to it, or a machine can consist of multiple computers with many physical components connected, acting as one unit.
 
 <div>
 {{< imgproc src="/viam/machine-components.png" alt="Machine components" resize="600x" class="aligncenter" >}}
```

Comment:
We've gotten a lot of feedback on this diagram, some of which is that people are confused by it. What do you think of replacing this instance of it with a screenshot of a config tab with some basic components configured (maybe board, motors, camera?)

Reply:
done